libmisc/salt.c

```diff
@@ -343,7 +343,6 @@ static /*@observer@*/const unsigned long BCRYPT_get_salt_rounds (/*@null@*/int *
 	 * because musl doesn't allow rounds >= 20.
 	 */
 	if (rounds > 19) {
-		/* rounds = B_ROUNDS_MAX; */
```

Comment:
Note that I know *I* am not smart enough that, if musl ever supports > 20 rounds, rounds should be set to B_ROUNDS_MAX.  Should the comment above at line 343 include that detail?

Reply:
I added the information in the comment